File: submissions/3-guys-1-code/challenge_6/challenge_6_min.java
static void f(final int cost) {
        final int tip = (cost < 2000 ? round(cost, 100) : cost < 5000 ? round(cost, 500) : roundNearest((int) (cost * 1.1))) - cost;
        // System.out.println("cost:" + cost + " tip:" + tip);
        System.out.println(tip == 0 || tip > 1500 ? "PAID BY CARD" : tip % 100 > 0 ? "KEEP THE CHANGE YOU FILTY ANIMAL" : "HERE IS " + (tip + cost));
    }

    static int round(final int a, final int b) {
        return ((a + b - 1) / b) * b;
    }

    static int roundNearest(final int a) {
        return ((a + 250) / 500) * 500;
    }

File: submissions/3-guys-1-code/challenge_6/challenge_6_min.rb
def r(a,b)
return ((a+b-1)/b)*b
end
def f(c)
@t=(c<2000?r(c,100):c<5000?r(c,500):(((c*1.1).floor+250)/500)*500)-c
puts @t==0||@t>1500?"PAID BY CARD":@t%100>0? "KEEP THE CHANGE YOU FILTY ANIMAL":"HERE IS #{@t+c}"
end